### PR TITLE
Fix ibc transfer and update scripts

### PR DESCRIPTION
**Pull request by piobab**

### CODE REVIEW

**Review by larry0x — APPROVED** [review]

**Comment by larry0x** [comment]
For context, `timeout_height` should be the block height of the receiver chain, not the sender chain. It's not possible to know the latest Mars Hub block height from inside our Osmosis contract, so here we only specify the `timeout_timestamp`, while leaving the `timeout_height` empty.

**Review by grod220 — APPROVED** [review]
Is there a way to write tests for this?

File: scripts/deploy/osmosis/config.ts [hunk 1]
```diff
@@ -2,58 +2,55 @@ import { DeploymentConfig, AssetConfig, OracleConfig } from '../../types/config'
 
 const axlUSDC = 'ibc/D189335C6E4A68B513C10AB227BF1C1D38C746766278BA3EEB4FB14124F1D858'
 const atom = 'ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2'
-const marsTestnet = 'ibc/ACA4C8A815A053CC027DB90D15915ADA31939FA331CE745862CDD00A2904FA17'
+const mars = 'ibc/ACA4C8A815A053CC027DB90D15915ADA31939FA331CE745862CDD00A2904FA17'
```
Comment (brimigs) [inline scripts/deploy/osmosis/config.ts:5]: mars testnet & mainnet denoms are different. Mars mainnet denom will be announced in 4 hours once IBC is enabled.

File: scripts/deploy/osmosis/config.ts [hunk 2]
```diff
@@ -62,37 +59,35 @@ export const osmosisTestMultisig: DeploymentConfig = {
...
 export const osmosisMainnet: DeploymentConfig = {
   chainName: 'osmosis',
   atomDenom: atom,
   baseAssetDenom: 'uosmo',
   chainId: 'osmosis-1',
   chainPrefix: 'osmo',
-  channelId: 'TBD',
-  marsDenom: marsMainnet,
-  rewardCollectorTimeoutBlocks: 100,
+  channelId: 'channel-2083',
```
Comment (brimigs) [inline scripts/deploy/osmosis/config.ts:78]: this is the testnet channel. I believe its going to be channel-1? We will know in a couple hours

File: scripts/deploy/osmosis/config.ts [hunk 3]
```diff
@@ -222,11 +215,6 @@ export const atomOracle: OracleConfig = {
   downtime_detector: { downtime: 'duration10m', recovery: 7200 },
 }
 
-export const osmoOracle: OracleConfig = {
```
Comment (brimigs) [inline scripts/deploy/osmosis/config.ts:225]: Was Bob able to test his fix for this?

File: scripts/deploy/osmosis/config.ts (hunk 2 above)
Comment (piobab) [reply scripts/deploy/osmosis/config.ts]: Mars -> Osmosis: channel-1
Osmosis -> Mars: channel-557

File: scripts/deploy/osmosis/config.ts (hunk 3 above)
Comment (piobab) [reply scripts/deploy/osmosis/config.ts:225]: Yes, he confirmed me. I removed it on testnet on Friday.

File: scripts/deploy/osmosis/config.ts (hunk 1 above)
Comment (piobab) [reply scripts/deploy/osmosis/config.ts]: Updated

**Comment by piobab** [comment]
I updated config to use correct mainnet params.

File: scripts/deploy/osmosis/config.ts [hunk 4]
```diff
@@ -62,29 +59,27 @@ export const osmosisTestMultisig: DeploymentConfig = {
...
+  channelId: 'channel-557',
   marsDenom: marsMainnet,
-  rewardCollectorTimeoutBlocks: 100,
   rewardCollectorTimeoutSeconds: 600,
   rpcEndpoint: 'https://rpc.osmosis.zone',
   safetyFundFeeShare: '0.5',
-  timeoutRevision: 1,
   deployerMnemonic: 'TO BE INSERTED AT TIME OF DEPLOYMENT',
   slippage_tolerance: '0.01',
   base_asset_symbol: 'OSMO',
```
Comment (dancreee) [inline scripts/deploy/osmosis/config.ts:85]: osmo1jklpvl3446z5qw58cvq8hqvthzjtsfvs9j65tq for mainnet should also be: osmo14w4x949nwcrqgfe53pxs3k7x53p0gvlrq34l5n

**Review by dancreee — CHANGES_REQUESTED** [review]

File: scripts/deploy/osmosis/config.ts (hunk 4 above)
Comment (dancreee) [reply scripts/deploy/osmosis/config.ts:85]: (multisigAddr that is)

Comment (dancreee) [reply scripts/deploy/osmosis/config.ts:85]: updated in https://github.com/mars-protocol/red-bank/pull/175/commits/2d8c5f110004399707cb0d00080f680c3033cc6c

**Review by dancreee — APPROVED** [review]

**Review by dancreee — APPROVED** [review]

**Pull request merged**